modular_citadel/code/game/machinery/vending.dm

```diff
@@ -43,18 +43,33 @@
 	vend_reply = "Have fun, you shameless pervert!"
 	products = list(
 				/obj/item/clothing/under/maid = 5,
-				/obj/item/clothing/under/stripper_pink = 5,
-				/obj/item/clothing/under/stripper_green = 5,
-				/obj/item/dildo/custom = 5
-				)
-	contraband = list(/obj/item/restraints/handcuffs/fake/kinky = 5,
 				/obj/item/clothing/neck/petcollar = 5,
-				/obj/item/clothing/under/mankini = 1,
-				/obj/item/dildo/flared/huge = 1
+				/obj/item/clothing/neck/petcollar/choker = 5,
+				/obj/item/clothing/neck/petcollar/leather = 5,
+				/obj/item/restraints/handcuffs/fake/kinky = 5,
+				/obj/item/clothing/glasses/sunglasses/blindfold = 4,
+				/obj/item/clothing/mask/muzzle = 4,
+				/obj/item/clothing/under/stripper_pink = 3,
+				/obj/item/clothing/under/stripper_green = 3,
+				/obj/item/dildo/custom = 5,
+				/obj/item/electropack/shockcollar = 3,
+				/obj/item/assembly/signaler = 3
+				)
+	contraband = list(
+				/obj/item/clothing/neck/petcollar/locked = 2,
+				/obj/item/key/collar = 2,
+				/obj/item/clothing/head/kitty = 3,
+				/obj/item/clothing/head/rabbitears = 3,
+				/obj/item/clothing/under/mankini = 2,
+				/obj/item/clothing/under/jabroni = 2,
+				/obj/item/dildo/flared/huge = 3,
+				/obj/item/reagent_containers/glass/bottle/crocin = 5,
+				/obj/item/reagent_containers/glass/bottle/camphor = 5
 				)
 	premium = list(
-				/obj/item/electropack/shockcollar = 3,
-				/obj/item/clothing/neck/petcollar/locked = 1
+				/obj/item/clothing/accessory/skullcodpiece/fake = 3,
+				/obj/item/clothing/under/gear_harness = 10,
+				/obj/item/reagent_containers/glass/bottle/hexacrocin = 10
 				)
```

Comment:
forgot hexacamphor

Reply:
I left hexacamphor out because it can apparently be used to intentionally grief and "sterilize" someone, but added it as a bottle because why not (especially when I added all the others as bottles anyway.)

Reply:
all of the bottles are used for intentional grief